spec/models/course_spec.rb

```diff
@@ -196,6 +196,56 @@
     end
   end
 
+  describe "#study_mode_description" do
+    specs = {
+      full_time: 'full time',
+      part_time: 'part time',
+      full_time_or_part_time: 'full time or part time',
+    }.freeze
+
+    specs.each do |study_mode, expected_description|
+      context study_mode.to_s do
+        subject { create(:course, study_mode: study_mode) }
+        its(:study_mode_description) { should eq(expected_description) }
+      end
+    end
+  end
+
+  describe "#description" do
```

Comment:
missing combo?
![image](https://user-images.githubusercontent.com/470137/53801056-14b99000-3f36-11e9-9d17-6de84c5a5a8b.png)

Reply:
There is test for 
`PGDE with QTS full time with salary`
`PGCE with QTS full time with salary`

but not 
`QTS full time with salary` (ie the pic above)

but in any case no reasons to hold this PR back to be honesty

was a simple sanity check